configuration-light.nix

```diff
@@ -86,7 +86,30 @@
     };
   };
 
+  # This is all needed to make resizing work inside the VirtualBox VM
   virtualisation.virtualbox.guest.enable = true;
+  systemd.user.services = let
+    vbox-client = desc: flags: {
+      description = "VirtualBox Guest: ${desc}";
+
+      wantedBy = [ "graphical-session.target" ];
+      requires = [ "dev-vboxguest.device" ];
+      after = [ "dev-vboxguest.device" ];
+
+      unitConfig.ConditionVirtualization = "oracle";
+
+      serviceConfig.ExecStart = "${config.boot.kernelPackages.virtualboxGuestAdditions}/bin/VBoxClient -fv ${flags}";
+    };
+  in {
+    virtualbox-resize = vbox-client "Resize" "--vmsvga";
+    virtualbox-clipboard = vbox-client "Clipboard" "--clipboard";
+  };
+
+  fileSystems."/" = {
+    device = "/dev/disk/by-label/nixos";
+    fsType = "ext4";
+  };
+  boot.loader.grub.device = "/dev/sda";
```

Comment:
light-vm didnt had this script and fs. why? havent tried it yet. so not sure it works currently and it works after